src/api.ts

```diff
@@ -0,0 +1,16 @@
+const API_URL = 'https://jsonplaceholder.typicode.com';
```

Comment:
minor: better to name this variable as BASE_URL instead of API_URL (just a common practice)